README.md

```diff
@@ -293,3 +293,10 @@ mvn install             # installs dependencies
 mvn test                # run tests
 mvn jacoco:report       # writes code coverage reports to ./target/site/jacoco
 ```
+
+# Environment Variables
+
+```bash
+TEST_API_KEY            # autocomplete_key for the tests
```

Comment:
Would it make sense to explicitly define which company/index to use here?

Reply:
That sounds good to me 👍